Addition of Events on Index Events Page (#115)
kubernetes-sigs/contributor-site: content/Events/_index.md

---
title: Events 

> description: Kubernetes Contributor Summits 

Author: Let the index page contain the introduction about the Contributor Summit and we can have a seperate sub-section for the past & future commits

---

> # What ?

Reviewer: ```suggestion
```
I'd just remove this altogether -- The text below will be a good enough intro.

Author: Sure

> This event brings together new and current Kubernetes contributors to connect and learn from one another. It is an opportunity for existing contributors to help shape the future of the project, and offers a welcoming space for new community members to learn, explore and put the contributor workflow to practice.

Reviewer: ```suggestion
Contributor Summits bring together new and current Kubernetes contributors to connect and learn from one another. It is an opportunity for existing contributors to help shape the future of the project, and offers a welcoming space for new community members to learn, explore and put the contributor workflow to practice.
```
As this is a page for multiple summits it should be plural.

> In some sense, the summit is a real-life extension of the community meetings and SIG meetings. There are three explicit goals:

Reviewer: ```suggestion
In some sense, Contributor Summits are a real-life extension of the community and SIG meetings. They aim to:
```

- Onboard new contributors to be productive in our community
- Send contributors home with more context, knowledge, and skills about the project

> - SIG-specific face-to-face collaboration

Reviewer: ```suggestion
- Facilitate SIG-specific face-to-face collaboration
```

> Contributor Summits mainly includes these three tracks :

Reviewer: ```suggestion
Contributor Summits often have three tracks:
```

> * The New Contributor Workshop is for those looking to find out how to become Kubernetes contributors;

Reviewer: ```suggestion
- **New Contributor Workshop** for those looking to find out how to become Kubernetes contributors
```

> * The Current Contributor Day is for attendees who are already contributing to Kubernetes, and want to learn more or do hands-on work at the Summit;

Reviewer: ```suggestion
- **Active Contributor Track** - Featuring technical discussions, unconference sessions, workshops, and more for those that are actively contributing to the Kubernetes project.
```

> * The Documentation Sprints are for community members who want to help with writing documentation or translating it.

Reviewer: ```suggestion
- The Documentation Sprints is for community members who want to help with writing or localizing documentation.
```

> - The **Documentation Sprint** is for community members who want to help with writing or localizing documentation.

Reviewer: This should be moved over to the contributor summit page. -- Like this would be better above the upcoming events section.

The landing page for events would probably be better to have a brief description of each sub-section or page under events.
